docs/tasks/active/20260731-cache-core-split-todo.md

```
@@ -0,0 +1,100 @@
+# Keep the review panel's prompt cache working at one sample per lens
+
+#607 cut the code-quality lenses to `samples: 1` and #610 did the same for
```

markdownlint-cli2 (0.23.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)